helm/templates/cloudwatch-agent-daemonset.yaml

```diff
@@ -12,9 +10,9 @@ spec:
   mode: daemonset
   serviceAccount: {{ template "cloudwatch-agent.serviceAccountName" . }}
   {{- if .Values.agent.config }}
-  config: {{ template "cloudwatch-agent.supplied-config" . }}
+  config: {{ include "cloudwatch-agent.modify-config" (merge (dict "Config" .Values.agent.config) . ) }}
   {{- else }}
-  config: {{ template "cloudwatch-agent.modify-default-config" . }}
+  config: {{ include "cloudwatch-agent.modify-config" (merge (dict "Config" .Values.agent.defaultConfig) . ) }}
```

Comment:
Just double checking, this dict with the `Config` key is just to access it in the func and wont actually exist in the final output of this right?